src/goto-bmc/goto_bmc_parse_options.cpp

```diff
@@ -0,0 +1,100 @@
+// Author: Fotis Koutoulakis for Diffblue Ltd.
+
+#include "goto_bmc_parse_options.h"
+
+#include <util/exit_codes.h>
+#include <util/message.h>
+
+#include <goto-programs/read_goto_binary.h>
+
+#include <libcprover-cpp/api_options.h>
+#include <libcprover-cpp/verification_result.h>
+
+#include "api.h"
+
+#include <iostream>
+#include <map>
+
+core_cbmc_only_parse_optionst::core_cbmc_only_parse_optionst(
+  int argc,
+  const char **argv)
+  : parse_options_baset(
+      CORE_CBMC_ONLY_OPTIONS,
```

Comment:
symex-ready

Reply:
These shouldn't be called `symex-ready` - they refer to the tool not the goto-binaries.

But they will need to be made consistent to the actual tool name, which we settled on `goto-bmc` (this is the name of the tool built). The references are artefacts from a previous change where we called the tool `corebmc` but settled on `goto-bmc` instead.